Applications/SimpleShot/task_runner.cpp

```diff
@@ -175,15 +233,161 @@ std::unique_ptr<ml::train::Model> createModel(const std::string &backbone,
 }
 } // namespace simpleshot
 
+/**
+ * @brief Check whether file exist or not
+ * @param file_name name of file
+ * @return bool
+ */
+
+bool isFileExist(std::string file_name) {
+  std::ifstream infile(file_name);
+  return infile.good();
+}
+
+/**
+ * @brief appends label in allLabels.txt file that maintains all added classes
+ * string labels
+ * @param label_path string label path of newly added classes
+ * @return int
```

Comment:
what does this int state?

Reply:
1 for new classes appended in allLabels.txt and 0 if no new classes are added to allLabels.txt

Reply:
Please Check now